armstrong.rb

```diff
@@ -0,0 +1,26 @@
+def armstrong_number(n)
```

Comment:
The `armstrong.rb` file must remain only the `week1` folder. This seems duplicated so please delete the file.